sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -251,22 +258,27 @@ class FileBrowserElement extends Polymer.Element {
   /**
    * Updates the breadcrumbs array and calls _fetchFileList.
    */
-  _currentPathChanged(_: string, oldValue: string) {
-    // On initialization, push the current path to path history
-    if (oldValue === undefined) {
-      this._pushNewPath();
+  _currentPathChanged() {
+    // Ignore inital '/'
+    if (this.currentPath.startsWith('/')) {
+      this.currentPath = this.currentPath.substr(1);
     }
 
-    this._currentCrumbs = this.currentPath.split('/');
+    this.$.breadCrumbs.crumbs = this._splitCurrentPath();
+
+    return this._fetchFileList();
+  }
+
+  _splitCurrentPath() {
+    const pathTokens = this.currentPath.split('/');
```

Comment:
You could add .filter((p) => p) here (or !!p, or p!='') instead of the if-statement on line 278